sys/clif/clif.c

```diff
@@ -299,10 +302,11 @@ ssize_t clif_get_attr(const char *input, size_t input_len, clif_attr_t *attr)
     if (scan_value) {
         /* iterate over value */
         while (pos < end) {
-            if (quoted) {
+            if (quotes == 1) {
```

Comment:
What if "quotes > 1" but `(*pos == '"' && *(pos - 1) != '\\')`.

Also, as a side concern: can it be possible here that `*(pos - 1)` accesses memory outside the input? Probably not as quotes needs to be at least one for that to happen (so `pos` must have advanced at least one char), but something like this should be commented IMHO.

Reply:
See 10de4690eb498ae5f324b702b3c6229b0b361ccd and eca7f72fbd80e0a7ae5d086f1d592140be9b2e63